docs/src/modules/java/pages/samples.adoc

```diff
@@ -14,7 +14,10 @@ Samples are available that demonstrate important patterns and abstractions. Thes
 |=======================
 | Description | Source download | Level
 | xref:java:shopping-cart/index.adoc[Build a shopping cart] | link:https://github.com/akka-samples/shopping-cart-quickstart[Github Repository] |Beginner
-| xref:java:ask-akka/index.adoc[AI agent that performs a RAG workflow] | link:https://github.com/akka-samples/ask-akka-agent[Github Repository] |Intermediate
+| xref:java:ask-akka/index.adoc[AI agent that performs a RAG workflow] using Langchain4J. Chat memory by using entity and view. | link:https://github.com/akka-samples/ask-akka-agent[Github Repository] |Intermediate
+| AI agent that creates personalized travel itineraries. Illustrates reliable interaction with a LLM using Langchain4J and a workflow. Entities are used for durable state of user preferences and generated trips. | link:https://github.com/akka-samples/ask-akka-agent[Github Repository] |Beginner
+| AI agent that leverages a LLM to process medical discharge summaries and assign tags, while also enabling human verification and comparative analysis. Interactions with OpenAI LLM using the OpenAI Java SDK and a workflow. | link:https://github.com/akka-samples/medical-tagging-agent[Github Repository] |Intermediate
```

Comment:
@aludwiko pls modify if you want to explain it in some other way

Reply:
looks good